frontend/src/context/ws-client-provider.tsx

```diff
@@ -173,7 +150,7 @@ export function WsClientProvider({
       sio.off("connect_failed", handleError);
       sio.off("disconnect", handleDisconnect);
     };
-  }, [enabled, token, ghToken, selectedRepository]);
+  }, [enabled, ghToken, selectedRepository]);
```

Comment:
Wouldn't we want the effect to trigger when `conversationId` changes instead of `token`?